### PR TITLE
Remove SonataIntlBundle dependency

### PR DESCRIPTION
**Pull request by franmomu**
## Subject

Ref https://github.com/sonata-project/SonataIntlBundle/issues/280

I am targeting this branch, because this is BC.

## Changelog

```markdown
### Added
- Support for Twig 3
### Removed
- Remove `sonata-project/intl-bundle` dependency
```

### CODE REVIEW

**Comment by core23** [comment]
There is no usage in Twig templates too?

**Comment by franmomu** [comment]
> There is no usage in Twig templates too?

You're right 😟

File: composer.json [hunk 1]
```diff
@@ -37,7 +36,8 @@
         "symfony/http-kernel": "^3.4 || ^4.2",
         "symfony/options-resolver": "^3.4 || ^4.2",
         "symfony/security-core": "^3.4 || ^4.2",
-        "twig/twig": "^1.35 || ^2.4"
+        "twig/intl-extra": "^3.0",
+        "twig/twig": "^2.12"
```
Comment (core23) [inline composer.json]:
```suggestion
        "twig/twig": "^2.12 || ^3.0"
```

Can we use this?

File: src/Resources/config/twig.xml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<container xmlns="http://symfony.com/schema/dic/services" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+    <services>
+        <service id="sonata.timeline.twig.intl_extension" class="Twig\Extensions\IntlExtension">
```
Comment (core23) [inline src/Resources/config/twig.xml]: This is needed?

File: composer.json (hunk 1 above)
Comment (franmomu) [reply composer.json]: I think so

File: src/Resources/config/twig.xml (hunk 2 above)
Comment (franmomu) [reply src/Resources/config/twig.xml]: When I've used them in a project, I had to add them as services. In [the docs](https://twig.symfony.com/doc/3.x/filters/format_date.html) it says:
```php
use Twig\Extra\Intl\IntlExtension;

$twig = new \Twig\Environment(...);
$twig->addExtension(new IntlExtension());
```
which I haven't check but I guess is what `twig.extension` tag does.

Comment (core23) [reply src/Resources/config/twig.xml]: Okay then. We could keep this for the stable release to stay BC. We should remove this service and update the installation instruction for the next major release.

**Comment by franmomu** [comment]
Since the minimum is now `4.3`, I required `twig/extra-bundle` which registers the IntlExtension.

**Comment by OskarStark** [comment]
Ups

**Comment by greg0ire** [comment]
Thanks @franmomu !

**Pull request merged**